server/shovel-config.ts

```diff
@@ -1,69 +1,95 @@
 import { makeConfig, toJSON } from '@indexsupply/shovel-config'
-import type { Source, Table, Integration } from '@indexsupply/shovel-config'
-import {
-  Problems,
-  Bodies,
-  Solver,
-  Dust,
-  ProblemMetadata,
-  BodyMetadata
-} from './contracts'
+import type {
+  Source,
+  Table,
+  Integration,
+  PGColumnType,
+  IndexStatment
+} from '@indexsupply/shovel-config'
+import { Problems, Bodies, Solver } from './contracts'
 import { ethers } from 'ethers'
+import { camelToSnakeCase } from './src/util'
 
-const mainnet: Source = {
+export type Chain = 'mainnet' | 'sepolia' | 'garnet' | 'base_sepolia'
+type KnownSource = Source & { name: Chain }
+
+const mainnet: KnownSource = {
   name: 'mainnet',
   chain_id: 1,
   url: 'https://ethereum-rpc.publicnode.com'
 }
 
-const sepolia: Source = {
+const sepolia: KnownSource = {
   name: 'sepolia',
   chain_id: 11155111,
-  url: 'https://rpc2.sepolia.org', // 'https://ethereum-sepolia-rpc.publicnode.com' rate limited
+  url: 'https://rpc2.sepolia.org',
```

Comment:
same wrt env var for rpc url, we have paid subscription so storing api key publicly could invite griefing

Reply:
ah my bad. yea will move to env